bin/setup

```diff
@@ -5,7 +5,6 @@ def setup!
     test "ruby -v" => ruby_version
     run  "gem install bundler --no-document --conservative"
     run  "bundle install"
-    run  "bin/yarn install" if File.exist?("yarn.lock")
```

Comment:
This shouldn't be removed as it's part of our setup script, and it's reasonable to be doing given the condition